pkg/componentinfo/component_info.go

```diff
@@ -83,6 +85,10 @@ func (ci *ComponentInfo) KubernetesAPIs() ([]string, error) {
 	return metav1.ExtractGroupVersions(groups), nil
 }
 
+func (ci *ComponentInfo) Namespace(name string) (*v1.Namespace, error) {
```

Comment:
Does getting the complete namespace make sense?

Also, should I add a comment to get rid of the golint error or is there another way of getting rid of it because the comment would be just adding some noise and not helping much.

Reply:
I think the noise is ok, go ahead and add the comment.

> Does getting the complete namespace make sense?

Probably not for this PR? I think we just want the status. We can always refactor later when we have the usecase to need the whole NS object.

Reply:
Done.